index.html

```diff
@@ -1,467 +1,600 @@
     <link
-        href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@200&family=Roboto:wght@300;700&family=Smooch+Sans:wght@100&display=swap"
-        rel="stylesheet" />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@200&family=Roboto:wght@300;700&family=Smooch+Sans:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
     <link href="fonts/InaiMathi.ttf" rel="stylesheet" />
+    <script defer src="./index.js"></script>
     <title>Document</title>
-</head>
+  </head>
 
-<body>
+  <body>
     <div class="body">
-        <header id="header">
-            <div class="wrapper">
-                <div class="wrapp">
-                    <div class="logo">
-                        <a>Alena Okolike</a>
-                    </div>
-                    <nav class="nav">
-                        <ul>
-                            <li><a href="#">Portfolio</a></li>
-                            <li><a href="#">About</a></li>
-                            <li><a href="#">Contact</a></li>
-                            <li><img src="./images/Union.png" alt="" /></li>
-                        </ul>
-                    </nav>
-                    <nav class="hamburger">
-                        <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-                            <path fill-rule="evenodd" clip-rule="evenodd"
-                                d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
-                                fill="white" />
-                        </svg>
-                    </nav>
-                </div>
+      <header id="header">
+        <nav class="mobile-nav">
+          <ul>
+            <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+            <li class="mobile-nav-link"><a href="#aboutme">About</a></li>
+            <li class="mobile-nav-link"><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+        <div class="wrapper">
+          <div class="wrapp">
+            <div class="logo">
+              <a>Alena Okolike</a>
```

Comment:
- [ **OPTIONAL** ] **`RECOMENDED:`** I think it would be better if you update your logo link by adding a `href="..."`  attribute in the `<a>` tag so it can be used to access the home section i.e `<a href="#">Alena Okolike</a>`